Library/TimerLabelTests.swift

```diff
@@ -0,0 +1,84 @@
+import Library
+import XCTest
+
+final class TimerLabelTests: XCTestCase {
+    private weak var timer: Timer!
+
+    override func setUp() {
+        // swiftlint:disable line_length
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/2791e71195857606bdc9349e0f6f10b8418f390d/Library/TimerLabelTests.swift#L8"></span>SwiftLint rule ‘line_length’ did not trigger a violation in the disabled region. Please remove the disable command.<br />
<code>superfluous_disable_command</code> <code>TimerLabelTests.swift:8</code></td>
    </tr>
  </tbody>
</table>